general/summary/cli.go

```diff
@@ -0,0 +1,229 @@
+package summary
+
+import (
+	"errors"
+	"fmt"
+	"github.com/jfrog/jfrog-cli-core/v2/artifactory/utils/commandsummary"
+	"github.com/jfrog/jfrog-cli/utils/cliutils"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/jfrog/jfrog-cli-core/v2/artifactory/utils"
+	commonCliUtils "github.com/jfrog/jfrog-cli-core/v2/common/cliutils"
+	coreConfig "github.com/jfrog/jfrog-cli-core/v2/utils/config"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/coreutils"
+	securityUtils "github.com/jfrog/jfrog-cli-security/utils"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+	"github.com/urfave/cli"
+)
+
+type MarkdownSection string
+
+const (
+	Security  MarkdownSection = "security"
+	BuildInfo MarkdownSection = "build-info"
+	Upload    MarkdownSection = "upload"
+)
+
+const (
+	JfrogCliSummaryDir = "jfrog-command-summary"
+	MarkdownFileName   = "markdown.md"
+)
+
+var markdownSections = []MarkdownSection{Security, BuildInfo, Upload}
+
+func (ms MarkdownSection) String() string {
+	return string(ms)
+}
+
+// Creates a final summary of recorded CLI commands that were executed on the current machine.
+// The summary is generated in Markdown format and saved in the root directory of JFROG_CLI_COMMAND_SUMMARY_OUTPUT_DIR.
```

Comment:
...saved in the root directory of JFROG_CLI_COMMAND_SUMMARY_OUTPUT_DIR.
-->
...saved in the directory stored in the JFROG_CLI_COMMAND_SUMMARY_OUTPUT_DIR environment variabke.